pkg/exec/devfile.go

```diff
@@ -11,8 +11,8 @@ import (
 	"github.com/pkg/errors"
 )
 
-// ExecuteDevfileBuildAction executes the devfile build command action
-func ExecuteDevfileBuildAction(client ExecClient, exec common.Exec, commandName string, compInfo adaptersCommon.ComponentInfo, show bool, machineEventLogger machineoutput.MachineEventLoggingClient) error {
+// ExecuteDevfileCommands executes the devfile init, build and test command actions
+func ExecuteDevfileCommands(client ExecClient, exec common.Exec, commandName string, compInfo adaptersCommon.ComponentInfo, show bool, machineEventLogger machineoutput.MachineEventLoggingClient) error {
```

Comment:
probably rename this `ExecuteDevfileCommandSynchronously` to
1. differentiate from `ExecuteDevfileRunAction`
2. init, build and test are all synchronous/blocking execution

Reply:
Updated